### DISCUSSION TITLE
Trouble with unsupported card

### DISCUSSION THREAD

**Discussion by shantanubindhani** [OpenSC/OpenSC#2809] [Q&A]
I have a custom card here, manufactured by Capricorn identity services( as told by my team lead)
the issue is that the pkcs11.h is unable to identify it's token or something, 

while using the command line tool(Opensc-tool) i get the following results : 

   Command                            |            Result
---------------------------------------+------------------------------------------
   opensc-tool --name             |            Unsupported card
---------------------------------------+------------------------------------------
   opensc-tool --atr                  |            ( ATR of the card)
---------------------------------------+------------------------------------------

**Comment by Jakuje** [comment]
what are you trying to achieve?

**Comment by Jakuje** [comment]
Closing as non-relevant as there is no useful information provided.
